src/hooks/offramp/useMainProcess.ts

```diff
@@ -129,9 +129,9 @@ export const useMainProcess = () => {
     setSigningPhase,
   ]);
 
-  useEffect(() => {
-    setOnSelectedNetworkChange(resetOfframpingState);
-  }, [setOnSelectedNetworkChange, resetOfframpingState]);
+  // useEffect(() => {
+  //   setOnSelectedNetworkChange(resetOfframpingState);
```

Comment:
I do not know why the act of passing this function triggers it, even though there is no network change.